### PR TITLE
The problem with provably

### PR DESCRIPTION
**Pull request by wkumari**
Creating PR so it is easier to comment

### CODE REVIEW

File: Versions/draft-ietf-dnsop-dns-zone-digest-14.txt [hunk 1]
```diff
@@ -715,11 +715,11 @@ Internet-Draft               DNS Zone Digest                October 2020
...
-       records in the zone.  This is done by examining locally
-       configured trust anchors, and, if necessary, querying for (and
-       validating) DS RRs in the parent zone.  For zones that are
-       provably insecure, or if DNSSEC validation is not performed,
-       digest verification continues at step 4 below.
+       records in the zone.  This is done by validating the zone's
+       DNSKEY RRSet and determining its DNSSEC state ([RFC4033]).
+       Validation uses locally configured trust anchors, and, if
+       necessary, querying for intermediate keys and DS records in
+       ancestor zones.
```
Comment (wkumari) [inline Versions/draft-ietf-dnsop-dns-zone-digest-14.txt:722]: "Note that this is normal DNSSEC behavior"?

File: Versions/draft-ietf-dnsop-dns-zone-digest-14.txt [hunk 2]
```diff
@@ -730,16 +730,22 @@ Wessels, et al.          Expires April 12, 2021                [Page 13]
...
+       existence (NSEC/NSEC3) records prove whether or not the apex
+       ZONEMD RRSet exists.  When it is proven to not exist, digest
+       verification cannot occur.  When it is proven to exist, but is
+       not found in the zone, digest verification MUST NOT be considered
+       successful.
+
+   3.  For zones whose DNSKEY RRSet is Secure, the SOA and ZONEMD RRSets
+       MUST have valid signatures.  If DNSSEC validation of the SOA or
+       ZONEMD records fails, digest verification MUST NOT be considered
+       successful.
```
Comment (wkumari) [inline Versions/draft-ietf-dnsop-dns-zone-digest-14.txt:748]: Perhaps we could just say that "Normal DNSSEC validation is performed on the SOA and ZONEMD. If the validation of either of these fails, digest verification MUST NOT be considered successful. If validation results in Insecure or Indeterminate (see RFCxxx), digest verification continues at step 4 below." ? It feels like we are unnecessarily describing "normal" DNSSEC behavior, which leads to people poking at it....

Comment (wessels) [reply Versions/draft-ietf-dnsop-dns-zone-digest-14.txt:748]: Yes I was thinking along the same lines earlier today.  Let me take another stab at writing it that way.

**Comment by wessels** [comment]
Not merging.  Went with the "the-problem-with-provably-2" branch which was less DNSSEC detailed and closer to the original.

**Pull request closed without merging**